src/pages/tailwind-config-demo.js

```diff
@@ -0,0 +1,167 @@
+          <h2 className = 'font-avenir text-h5 text-orange-primary'>
+            colors
+          </h2>
+          <section className = 'flex mb-4 -mx-2'>
+            <div className = 'lg:w-1/5 px-2'>
+              <div className = 'flex w-full bg-orange-100'>
+                <div className = 'bg-orange-primary w-1/2'>
+                </div>
+                <div className = 'flex-column w-1/2'>
+                  <div className = 'h-10 bg-orange-100'>
+
+                  </div>
+                  <div
+                    className = 'h-10 bg-orange-200'
+                  >
+
+                  </div>
+                  <div className = 'h-10 bg-orange-300'>
+
+                  </div>
+                  <div className = 'h-10 bg-orange-400'>
+
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className = 'lg:w-1/5 px-2'>
+              <div className = 'flex w-full bg-dark-blue-100'>
+                <div className = 'bg-dark-blue-primary w-1/2'>
+
+                </div>
+                <div className = 'flex-column w-1/2'>
+                  <div className = 'h-10 bg-dark-blue-100'>
+
+                  </div>
+                  <div
+                    className = 'h-10 bg-dark-blue-200'
+                  >
+
+                  </div>
+                  <div className = 'h-10 bg-dark-blue-300'>
+
+                  </div>
+                  <div className = 'h-10 bg-dark-blue-400'>
+
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className = 'lg:w-1/5 px-2'>
+              <div className = 'flex w-full bg-gray-100 h-full'>
+                <div className = 'bg-gray-primary w-1/2 h-full'>
+
+                </div>
+                <div className = 'flex-column w-1/2'>
+                  <div className = 'h-20 bg-gray-100'>
+
+                  </div>
+                  <div
+                    className = 'h-20 bg-gray-200'
+                  >
+
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className = 'lg:w-1/5 px-2'>
+              <div className = 'flex w-full bg-orange-100 h-full'>
+                <div className = 'bg-light-green-primary w-1/2 h-full'>
+
+                </div>
+                <div className = 'flex-column w-1/2'>
+                  <div className = 'h-20 bg-light-green-100'>
+
+                  </div>
+                  <div
+                    className = 'h-20 bg-light-green-200'
+                  >
+
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className = 'lg:w-1/5 px-2'>
+              <div className = 'flex w-full bg-orange-100 h-full'>
+                <div className = 'bg-dark-green w-full h-full'>
+
+                </div>
+              </div>
+            </div>
+          </section>
+         
+          <section>
+
+          </section>
```

Comment:
This `<section>` is empty, any reason?

Reply:
Not really. Wanted to create another component based on the styleguide but the once created already solved the issue. I will remove it